### PR TITLE
Fixed: Updated Readme.md Issue #9

### PR DESCRIPTION
**Pull request by VivanVatsa**
## Related Issue

- Revamped readme.md for tech stack issue #9 

Closes: #9 

#### Describe the changes you've made

- changes made in the technology stack on the readme.md

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

## Screenshots

|        Original         |          Updated           |
| :---------------------: | :------------------------: |
| **original screenshot** | <b>updated screenshot </b> |

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,16 +12,19 @@
...
-This is a Machine Learning Web Application which can predict three diaseases i.e. Diabetes, Heart disease and Cancer. The datasets have been taken from Kaggle. And overall analysis and model building for the diseases have been done in respective Jupyter Notebooks.
+This is a Machine Learning Web Application which can predict three diseases i.e. Diabetes, Heart disease and Cancer. The datasets have been taken from Kaggle. And overall analysis and model building for the diseases have been done in respective Jupyter Notebooks.
 
 - [Datasets](https://github.com/kritikaparmar-programmer/HealthCheck/tree/main/Datasets)
 
 ## 👩‍💻 Technology Stack
 
-- [Flask](https://github.com/pallets/flask)
+#### **Tools**
+- [Python](https://www.w3schools.com/python/python_intro.asp)
```
Comment (kritikaparmar-programmer) [inline README.md:22]: Change Python link to - https://www.python.org/

**Review by kritikaparmar-programmer — CHANGES_REQUESTED** [review]

**Comment by kritikaparmar-programmer** [comment]
Also, you have done changes to the main branch of your local repo

**Comment by VivanVatsa** [comment]
yes, I have committed to the main brain of my local repo, do i have to commit the changes in this repo itself?

**Comment by VivanVatsa** [comment]
fixed with requested changes @kritikaparmar-programmer

**Comment by kritikaparmar-programmer** [comment]
OK @VivanVatsa merging the pr. Keep contributing :)

**Pull request merged**